src/Features/OnboardingTasks/TaskLists.php

```diff
@@ -99,8 +99,9 @@ public static function maybe_add_default_tasks() {
 
 		self::add_list(
 			array(
-				'id'    => 'extended',
-				'title' => __( 'Things to do next', 'woocommerce-admin' ),
+				'id'      => 'extended',
+				'title'   => __( 'Things to do next', 'woocommerce-admin' ),
+				'sort_by' => array( 'isComplete', 'level' ),
```

Comment:
This feels a bit odd to me since we're just passing information to the server to pass to the client to sort.

Instead of a `sort_by` param this might be a good time to introduce chaining for these functions:

`Task::get_list( 'extended' )->sort_by( ... )->get_json()`

Reply:
Or as an alternative to chaining, we could consider allowing sorting of the original list:

```php
$list = Task::get_list( 'extended' );
$list->sort_by( ... );
$list->get_json();
```